<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationContact</class>
 <widget class="QWidget" name="CreationContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="NomContact"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="LabelNomContact">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="LabelPrenomContact">
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PrenomContact"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LabelEntrepriseContact">
      <property name="text">
       <string>Entreprise</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EntrepriseContact"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="LabelEmailContact">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="EmailContact"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="LabelUriContact">
      <property name="text">
       <string>Uri Photo</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="uriPhotoContact">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="LabelTelephoneContact">
      <property name="text">
       <string>Telephone</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="TelephoneContact"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BoutonConfirmerCreationContact">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Créer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BoutonAnnulerCreationContact">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 20pt &quot;Cantarell&quot;;
</string>
   </property>
   <property name="text">
    <string>Créer un contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BoutonSelectionPhoto">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sélectionner une photo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
